nuxeo-retention/src/main/java/org/nuxeo/retention/listeners/RetentionDocumentEventListener.java

```diff
@@ -59,7 +60,8 @@ public boolean acceptEvent(Event event) {
             log.trace("RetentionManager not started yet?!");
             return false;
         }
-        return retentionManager.getAcceptedEvents().contains(event.getName());
+        List<String> acceptedEvents = retentionManager.getAcceptedEvents();
+        return acceptedEvents != null ? acceptedEvents.contains(event.getName()) : false;
```

Comment:
Wouldn't it be better to fix `RetentionManager.getAcceptedEvents` instead to never return `null`?

Reply:
It could, but I'd rather try to be resilient to an NPE here

Reply:
The thing is, it's a very bad pattern to allow a API that returns a list to also return null. We've had very bad habits for a long time on this.

Reply:
ok